asyncapi/asgardeo/Package.md

```diff
@@ -1,7 +1,7 @@
-Listen to Asgardeo Events API from Ballerina
+Listen to [Asgardeo Events API](https://wso2.com/asgardeo/docs/references/asgardeo-events/) from Ballerina
 
 ## Package overview
-The `ballerinax/trigger.asgardeo` is a [Ballerina](https://ballerina.io/) trigger for Asgardeo.
+The `ballerinax/trigger.asgardeo` is a [Ballerina](https://ballerina.io/) trigger for [Asgardeo](https://wso2.com/asgardeo/).
```

Comment:
Under Compatibility, the Slack link should be changed to Discord.

Reply:
Addressed with https://github.com/ballerina-platform/asyncapi-triggers/commit/fcd92a802f674aec9896fac8bc36af872d2f40c5